src/types/Checkbox.ts

```diff
@@ -0,0 +1,15 @@
+import { BaseType } from "./Base";
+
+export enum CheckboxSize {
+  small = "small",
+  default = "medium",
+}
+
+export interface CheckboxType extends BaseType {
+  value?: boolean;
+  onChange?: (checked: boolean) => any | void;
+  required?: boolean;
+  size?: CheckboxSize;
+  disabled?: boolean;
+  intermediate?: boolean;
+}
```

Comment:
It's okay for the context of this PR, but it overlaps at 95% with `Switch` type declaration. I'm creating another issue to merge all `boolean` input types (`Checkbox` and `Switch` atm) and all `textual` input types (`InputNumber and InputText` atm). 

`boolean` and `textual` share some common `Input` enums and properties, but each category has its own characteristics.

Reply:
Created dedicated issue for this, see #74